gensim/models/phrases.py

```diff
@@ -133,19 +133,32 @@ def __init__(self, sentences=None, min_count=5, threshold=10.0,
         `delimiter` is the glue character used to join collocation tokens, and
         should be a byte string (e.g. b'_').
 
+        `recode_to_utf8` is an optional parameter (default True) for any2utf8 conversion of input sentences
```

Comment:
How about

*By default, the input sentences will be internally encoded to UTF-8. Set `recode_to_utf8=False` to skip this recoding step. Then the Phrases object may use more memory (unless your sentences are already bytestrings), but will be much faster to train (~2x faster).*